Firestore/core/src/firebase/firestore/remote/connectivity_monitor_noop.cc

```diff
@@ -29,7 +29,7 @@ using util::AsyncQueue;
 // build doesn't break on platforms which don't yet implement
 // `ConnectivityMonitor`.
 std::unique_ptr<ConnectivityMonitor> ConnectivityMonitor::Create(
-      const std::shared_ptr<AsyncQueue>& worker_queue) {
+    const std::shared_ptr<AsyncQueue>& worker_queue) {
```

Comment:
It is corrected by `./scripts/style.sh` though it should not.  I think, it's better to revert this change.

Reply:
Never mind, probably it is a legit formatting issue to fix.